src/index.js

```diff
@@ -33,8 +33,8 @@ i18n.init().then(() => {
   ReactDOM.render(
   <Provider store={store}>
     <App />
-  </Provider>, document.getElementById('root'));
-})
+  </Provider>, document.getElementById('root')
```

Comment:
Does the app still work? I needed the brace and brackets or vscode gave me an error